leaf-ui/js/displayAnalysis.js

```diff
@@ -239,4 +252,147 @@ function updateHistory(currentAnalysis){
 
     var log = new logObject(currentAnalysis, 0);
     historyObject.allHistory.push(log);
-}
\ No newline at end of file
+}
+
+/**
+ * Function to set up the initial config
+ */
+function addFirstAnalysisConfig(){
+    $(".log-elements").css("background-color", "");
+    $(".result-elements").css("background-color", "");
+    var id = "Configuration1"
+    currAnalysisConfig = new AnalysisConfiguration(id, analysisRequest);
+    analysisMap.set(id, currAnalysisConfig);
+    // TODO: Find better way to preserve original default from model
+    // Currently necessary for User Assignments List preservation
+    defaultUAL = currAnalysisConfig.userAssignmentsList;
+    console.log(analysisRequest.userAssignmentsList);
+    var buttonLabel = currAnalysisConfig.id + "-button";
+    var label = currAnalysisConfig.id + "-dropdown";
+    $("#analysis-sidebar").append(
+        '<button class="log-elements" id="'+currAnalysisConfig.id+'" style="padding: 12px; font-size: 16px; border: none; outline: none; background-color:#A9A9A9">' + currAnalysisConfig.id + '</button><div style="position:absolute; display:inline-block"><button class="dropdown" id= "'+buttonLabel+'" style="padding: 12px; font-size: 16px; height: 42px; border: none; outline: none;"><i class="fa fa-caret-down fa-2x" style="cursor:pointer;"></i></button>'
+        + '</div><div class = "dropdown-container" id="'+label+'"></div>'
+      );
+
+    console.log(analysisMap)
+}
+
+/**
+ * Adds a new analysisConfig to the sidebar
+ */
+function addAnalysisConfig(){
+    $(".log-elements").css("background-color", "");
+    $(".result-elements").css("background-color", "");
+    // Update current config with current analysisRequest and set the udpated config in map
+    currAnalysisConfig.updateAnalysis(analysisRequest);
+    analysisMap.set(currAnalysisConfig.id, currAnalysisConfig);
+
+    // Figure out number of new config, name and create it, and then add it to the map
+    var numElements = analysisMap.size;
+    var id = "Configuration" + (numElements+1).toString()
```

Comment:
I don't think we need the `numElements` var
```suggestion
    var id = "Configuration" + (analysisMap.size+1).toString()
```

Reply:
Done thank you!